oggsound/include/OgreOggSoundManager.h

```diff
@@ -638,6 +642,10 @@ namespace OgreOggSound
 		static boost::recursive_mutex mSoundMutex;
 		static boost::recursive_mutex mResourceGroupNameMutex;
 #	endif
+*/
+		OGRE_WQ_MUTEX(mMutex);
```

Comment:
we must use the _WQ_ variants as the other ones are noops with OGRE_THREAD_SUPPORT=3

Reply:
Oh, I see. Thanks

Reply:
@paroj I pushed a new version with your fixes, now the code compiles but I don't know how well it works. Haven't been able to test it yet.

Are you able to test this with POCO, Boost and TBB?

This is only for streaming right?

Reply:
no idea.. never used threading here

Reply:
> Are you able to test this with POCO, Boost and TBB?

I would just leave it after porting to ogre threading. We can then wait until someone with an actual application using this comes along to properly test it.

Reply:
OK